src/pyfaktory/client.py

```diff
@@ -261,6 +262,9 @@ def _send_and_receive(self, command: str) -> str:
             return msg
 
     def _raise_error(self, faktory_response):
+        if not faktory_response:
```

Comment:
When does this happen?

Reply:
- I don't know why the response is empty, but I definitely encountered something. 


![image](https://github.com/user-attachments/assets/37a1d70b-d9d8-4254-acfe-c33ae4ff06de)

Reply:
You received an empty response for which command?

Reply:
![image](https://github.com/user-attachments/assets/28fc1e55-1767-4985-bc41-b67ea22769e6)

Reply:
> You received an empty response for which command?

Idle heartbeat waiting status

Reply:
According to FWP, `BEAT` command will never have an empty response (see [this](https://github.com/contribsys/faktory/blob/main/docs/protocol-specification.md#beat-command)) and I can't find in which case an empty response is sent.

If it is not a bug elsewhere in the code, and if this is not received for an invalid command, I think that you should open an issue in the [Faktory server repo](https://github.com/contribsys/faktory).

Reply:
maybe this [https://github.com/contribsys/faktory/issues/41](https://github.com/contribsys/faktory/issues/41)?

Reply:
i guess when client disconnect ,but  self fetch is executing?

![image](https://github.com/user-attachments/assets/7c071ec2-f4dd-4320-9db4-573105bcc1c6)
![image](https://github.com/user-attachments/assets/28ba34d0-89c7-4a31-a500-bcf485135e87)

Reply:
Ok, I see. Then I guess the fix should be [this](https://github.com/ghilesmeddour/faktory_worker_python/pull/14). What do you think ?

Reply:
ok  i will  try.